### PR TITLE
Add crashing-bitcoin-core-with-niklas-gogges-fuzzamoto review to bitcoinplusplus/insider-edition

### PR DESCRIPTION
**Pull request by liz831**
This PR adds [crashing-bitcoin-core-with-niklas-gogges-fuzzamoto](https://youtu.be/_TsK_3bYpu8) transcript review to the bitcoinplusplus/insider-edition directory.

### CODE REVIEW

File: bitcoinplusplus/insider-edition/crashing-bitcoin-core-with-niklas-gogges-fuzzamoto.md [hunk 1]
```diff
@@ -1,38 +1,36 @@
...
+source_file: 'https://youtu.be/_TsK_3bYpu8'
+summary: 'In this Bitcoin++ Insider Edition interview, niftynei sits down with Niklas Gögge (developer at Brink) to discuss Fuzzamoto, a coverage-guided fuzzing framework built to test Bitcoin full node implementations — currently Bitcoin Core, BTCD, and Libitcoin — by running the actual production daemon and exercising it through its external P2P and RPC interfaces rather than individual functions. The interview covers fuzzing fundamentals (coverage maps, corpus mutation, dictionary hints) and includes a live demo where Fuzzamoto, running across 32 parallel VMs, rediscovers a historic divide-by-zero bug in Bitcoin Core''s Bloom filter code triggered by a specific filterload/filteradd message sequence, showcasing how automated fuzzing removes human bias and surfaces multi-step edge cases that unit tests typically miss.'
 ---
 
 Speaker 0: 00:00:00
 
-I mean, I can just like start blabbing about the tool and like...
+Speaker 1 (00:00:04):
+
+"I mean, I can just start talking about the tool.
```
Comment (dillamondgoat) [inline bitcoinplusplus/insider-edition/crashing-bitcoin-core-with-niklas-gogges-fuzzamoto.md:25]: The original sentence was correct. Did you watch the video to verify the changes?"

File: bitcoinplusplus/insider-edition/crashing-bitcoin-core-with-niklas-gogges-fuzzamoto.md [hunk 2]
```diff
@@ -1,38 +1,36 @@
...
 
-Speaker 1: 00:00:04
+So, let's begin. Hey, I'm Nifty. You're Niklas, and you work at Brink. What are you working on these days at Brink, Niklas?"
```
Comment (dillamondgoat) [inline bitcoinplusplus/insider-edition/crashing-bitcoin-core-with-niklas-gogges-fuzzamoto.md:27]: The speaker didn't say "let's begin"
